.github/validate-pr/index.js

```diff
@@ -53,7 +53,7 @@ async function run () {
       per_page: 100
     })
     if (data.length > 0) {
-      files.push(...data.map(entry => entry.filename))
+      files.push(...data.filter(entry => entry.status !== 'deleted').map(entry => entry.filename))
```

Comment:
Validation fails because it tries to validate `indices.unfreeze`. This change should fix it, but it needs to be merged before it has any effect.